diff --git a/gio/gdesktopappinfo.c b/gio/gdesktopappinfo.c
index 343f30949..48467d189 100644
--- a/gio/gdesktopappinfo.c
+++ b/gio/gdesktopappinfo.c
@@ -2700,6 +2700,7 @@ prepend_terminal_to_vector (int          *argc,
     const char *exec_arg;
   } known_terminals[] = {
     { "xdg-terminal-exec", NULL },
+    { "ptyxis", "--" },
     { "kgx", "-e" },
     { "gnome-terminal", "--" },
     { "mate-terminal", "-x" },
diff --git a/gio/tests/desktop-app-info.c b/gio/tests/desktop-app-info.c
index c16e15a71..e9bd06031 100644
--- a/gio/tests/desktop-app-info.c
+++ b/gio/tests/desktop-app-info.c
@@ -1401,6 +1401,8 @@ get_terminal_divider (const char *terminal_name)
 {
   if (g_str_equal (terminal_name, "xdg-terminal-exec"))
     return NULL;
+  if (g_str_equal (terminal_name, "ptyxis"))
+    return "--";
   if (g_str_equal (terminal_name, "kgx"))
     return "-e";
   if (g_str_equal (terminal_name, "gnome-terminal"))
@@ -1923,6 +1925,7 @@ main (int   argc,
   guint i;
   const gchar *supported_terminals[] = {
     "xdg-terminal-exec",
+    "ptyxis",
     "kgx",
     "gnome-terminal",
     "mate-terminal",
