--- a/src/protocols/wlr_screencopy.rs
+++ b/src/protocols/wlr_screencopy.rs
@@ -150,7 +150,7 @@
     pub size: Size<i32, BufferCoords>,
     /// Available SHM formats, and rowstride for that format
     pub shm: Vec<(wl_shm::Format, u32)>,
-    #[cfg(any(feature = "winit", feature = "x11"))]
+    #[cfg(any(feature = "winit", feature = "x11", feature = "udev"))]
     /// Available linux-dmabuf formats, if any
     pub dma: Vec<Fourcc>,
 }
