From e50bb4482d8171b3cad748706b930684e0edccae Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 29 Oct 2020 18:21:06 +0100
Subject: [PATCH] main: Dump stack on segfaults by default

---
 src/main.c | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/main.c b/src/main.c
index af4fa8597e..03772ec7fb 100644
--- a/src/main.c
+++ b/src/main.c
@@ -46,6 +46,7 @@ static char *script_path = NULL;
 #define DBUS_REQUEST_NAME_REPLY_PRIMARY_OWNER 1
 #define DBUS_REQUEST_NAME_REPLY_ALREADY_OWNER 4
 
+#define DEFAULT_SHELL_DEBUG SHELL_DEBUG_BACKTRACE_SEGFAULTS
 enum {
   SHELL_DEBUG_BACKTRACE_WARNINGS = 1,
   SHELL_DEBUG_BACKTRACE_SEGFAULTS = 2,
@@ -364,8 +365,11 @@ shell_init_debug (const char *debug_env)
     { "backtrace-segfaults", SHELL_DEBUG_BACKTRACE_SEGFAULTS },
   };
 
-  _shell_debug = g_parse_debug_string (debug_env, keys,
-                                       G_N_ELEMENTS (keys));
+  if (debug_env)
+    _shell_debug = g_parse_debug_string (debug_env, keys,
+                                         G_N_ELEMENTS (keys));
+  else
+    _shell_debug = DEFAULT_SHELL_DEBUG;
 }
 
 static GLogWriterOutput
-- 
2.51.1

