--- a/meson.build	2026-03-13 01:57:39.782234720 +0530
+++ b/meson.build	2026-03-13 01:57:39.789999259 +0530
@@ -193,16 +193,8 @@
   required: false
 )
 
-# Needs to be a subproject since tecla does not declare dependencies
-tecla_is_subproject = not tecla_dep.found()
-if tecla_is_subproject
-  tecla = subproject(
-    'tecla',
-    default_options: [
-      'werror=false',
-    ],
-  )
-endif
+# Subproject fallback disabled for EL10 bootstrap
+tecla_is_subproject = false
 
 subproject('libgxdp',
   default_options: [
