From 455c98d429a93c385b535e695b908913a8a16295 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Tue, 14 Jul 2026 18:51:20 -0500
Subject: [PATCH] Make service a display manager

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 debian/cosmic-greeter.service | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/debian/cosmic-greeter.service b/debian/cosmic-greeter.service
index 278e591..70dcad3 100644
--- a/debian/cosmic-greeter.service
+++ b/debian/cosmic-greeter.service
@@ -17,6 +17,5 @@ RestartSec=1
 StartLimitBurst=5
 StartLimitInterval=30
 
-# Managed by debconf
-#[Install]
-#Alias=display-manager.service
+[Install]
+Alias=display-manager.service
-- 
2.54.0

